test/e2e/minscale_readiness_test.go

```diff
@@ -140,6 +155,11 @@ func TestMinScale(t *testing.T) {
 	if lr, err := waitForDesiredScale(clients, newServiceName, lt(minScale)); err != nil {
 		t.Fatalf("The revision %q scaled to %d > %d after not being routable anymore: %v", newRevName, lr, minScale, err)
 	}
+
+	revision, err = clients.ServingClient.Revisions.Get(context.Background(), newRevName, metav1.GetOptions{})
```

Comment:
Could this be flaky if sometimes the revision hasnt quite finished reconciling the PA status? (ie. since I think we're not waiting for that reconcile to complete in waitForDesiredScale might it be possible for waitForDesiredScale to return before ActualScale has bubbled up?)